ckan/post_install_functions.sh

```diff
@@ -1,13 +1,21 @@
 install_standard_ckan_extension_github() {
     REPO_NAME="${1}"
-    EGG="${2}"
+    BRANCH=""
+    # EGG is part of REPO_NAME
```

Comment:
This is not always true

e.g. 
https://github.com/ViderumGlobal/ckan-cloud-docker/blob/master/.docker-compose.datagov-theme.yaml#L22